schema.graphql

```diff
@@ -70,6 +75,35 @@ type Dataset {
   status: JobStatus
   inputPath: String
   uploadDateTime: String
+
+  access: AccessRights!
+}
+
+type UserProfile {
```

Comment:
@lomereiter do we need Person type now?

Reply:
@intsco we need to replace `Person` with `UserProfile` but we either need to change the metadata schema (to replace name/surname with just full name) or `UserProfile` (to keep name/surname).

Reply:
@lomereiter using full name makes more sense now I think

Reply:
@intsco @andy-d-palmer ok, I suggest that we keep `principalInvestigator` for simplicity, but make it by definition the same as `group.manager` (where `group` is the group of that dataset - can be one-person group - but the rest of group information is not exposed to the public)